.gitignore

```diff
@@ -132,6 +132,7 @@ gen/
 .gradle/
 build/
 */build/
+android/app/gradle*
```

Comment:
It may not be. The only way I could wrestle android studio into being able
to navigate/compile the project was to allow it to install a gradle wrapper
at `android/app`.

Separately, it would be great if we had developer setup instructions for
Android Studio, in this repo.

On Tue, Aug 30, 2016 at 11:04 AM, Geoffrey Goh <notifications@github.com>
wrote:

> In .gitignore
> <https://github.com/Microsoft/react-native-code-push/pull/491#discussion_r76847599>
> :
>
> > @@ -132,6 +132,7 @@ gen/
> >  .gradle/
> >  build/
> >  */build/
> > +android/app/gradle*
>
> Why was this addition needed?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Microsoft/react-native-code-push/pull/491/files/f6ea4ec00d8528c665d76585154e0c064213a724#r76847599>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGnzXGro5WrB2BC_jVjp_SoV2UrNPMU3ks5qlHCfgaJpZM4JtcJe>
> .
>